src/repository/screens/repository.screen.js

```diff
@@ -39,6 +40,7 @@ const mapStateToProps = state => ({
   starred: state.repository.starred,
   forked: state.repository.forked,
   subscribed: state.repository.subscribed,
+  hasRepoExist: state.repository.hasRepoExist,
```

Comment:
Shall we keep `hasRepoExist` and `error` together? Or either is enough?

Reply:
Actually i want to remove `error`. But as you said before, you said maybe it will be used in future.

So, will me to delete that?

Reply:
I prefer to keep `error`, because `error` is needed for Toast, while `hasRepoExist` can be replaced by `!error`. What do you think about?

Reply:
@chinesedfan i think `hasRepoExist` is only to set state if repo exist or not, but `error` is to get value whatever it is not only about repo exist or not. I think both are used.

Reply:
Yes, there are tricky differences between them. It requires us to handle these 2 fields more carefully.